kpet/data.py

```diff
@@ -88,6 +88,59 @@ def __init__(self, data):
             data
         )
 
+    def get_case(self, name):
+        """
+        Get a test case by name.
+
+        Args:
+            name:   Name of the test case to get.
+
+        Returns:
+            The matching test case, or None if not found.
+        """
+        for case in self.cases:
+            if case.name == name:
+                return case
+        return None
+
+    def match_case_set(self, src_path_set):
+        """
+        Return test cases responsible for testing any files in a set.
+
+        Args:
+            src_path_set:   A set of source file paths to match cases against,
+                            or an empty set for all source files.
```

Comment:
Not sure I understand what you mean. This is an argument, we pass a set of source files to the function so it would return test cases which cover at least one of them. We pass an empty set to signify all source files, so all cases would match. We are not able to pass the actual set of all files.

Reply:
I see. Marking as resolved conversation.